### DISCUSSION TITLE
Domain does not resolve to the GitHub Pages server. For more information, see documentation (NotServedByPagesError).

### DISCUSSION THREAD

**Discussion by pauloviola16** [community/community#78045] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have this error when I try to register my custom domain, I have already configured my DNS and CNAME on hostinger and it still doesn't work.

**Comment by erickdc7** [comment]
The "NotServedByPagesError" you're encountering indicates that your custom domain is not resolving to the GitHub Pages server. This can happen for several reasons, including DNS propagation delays or misconfiguration. Here are some steps to troubleshoot the issue:

- **DNS Propagation**: It can take up to 48 hours for DNS changes to fully propagate. If you've recently made changes, you may need to wait longer.
- **DNS Configuration**: Double-check your DNS settings to ensure that the A records point to GitHub's IP addresses and that the CNAME record for `www` points to your GitHub Pages URL.
- **GitHub Settings**: Verify that you've entered the custom domain correctly in your repository's GitHub Pages settings and that you've committed a `CNAME` file with your custom domain name at the root of your repository.
- **Check with Hostinger**: Since you're using Hostinger, ensure that the DNS records are correctly set up on their platform. You may need to contact Hostinger support for assistance.

If you've checked all the above and the issue persists, you can refer to GitHub's [documentation on setting up a custom domain with GitHub Pages](^[1](https://stackoverflow.com/questions/54059217/how-to-fix-domain-does-not-resolve-to-the-github-pages-server-error-in-github)^) for more detailed instructions and troubleshooting tips. Remember, DNS changes can take time, so patience is often key. If necessary, reach out to GitHub support for further help assistant.
